pkg/controller/provider/container/ova/model.go

```diff
@@ -228,3 +231,36 @@ func (r *DiskAdapter) GetUpdates(ctx *Context) (updates []Updater, err error) {
 	}
 	return
 }
+
+type StorageAdapter struct {
+	BaseAdapter
+}
+
+func (r *StorageAdapter) GetUpdates(ctx *Context) (updates []Updater, err error) {
+	return
+}
+
+// List the collection.
+func (r *StorageAdapter) List(ctx *Context, provider *api.Provider) (itr fb.Iterator, err error) {
+	storageList := []Storage{}
+	storageName := fmt.Sprintf("Provider %s storage", provider.Name)
```

Comment:
how about `Fake storage for %s` ?

Reply:
umm how about `Dummy Storage for %s` or `Mock storage for %s`? personally I think fake sounds a bit off

Reply:
it reminds me of [a post by Martin Fowler where he listed fakes, mocks, dummies, stubs, ...](https://martinfowler.com/articles/mocksArentStubs.html) - I don't think 'mock' is not the right term to use here but 'dummy' sounds good, it aligns with `Dummy objects are passed around but never actually used. Usually they are just used to fill parameter lists.` and it doesn't suggest that it's not production-ready like fakes

Reply:
nice, so dummy it is:)